src/drawing/DrawingCreator.tsx

```diff
@@ -21,6 +21,8 @@ export const defaultStroke = {
     size: 4,
 };
 
+const MIN_SIZE = 4;
```

Comment:
It seems like the default width is duplicated here. Should we use `stroke.width` as the minimum value, instead?

Reply:
Good idea